FS.common.js

```diff
@@ -25,6 +26,21 @@ var _downloadFile = Promise.promisify(RNFSManager.downloadFile);
 var _uploadFiles = Promise.promisify(RNFSManager.uploadFiles);
```

Comment:
@wootwoot1234 — I did have an issue with promisify uploadFiles, so I need to verify this before claiming it's good to go.